packages/dashboard/src/components/lists/Watchlist.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { tradeActions } from '../../redux/actions'
+
+import {
+  Alert,
+  Container,
+  Row,
+  Col,
+  InputGroup,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Card,
+  CardHeader,
+  CardBody,
+} from 'shards-react'
+
+
+
+
+const Watchlist = ({
+  accounts,
+  user,
+  historicQuotes,
+  globalSelectedAccount,
+}) => {
+  
+
+  
+
+  return (
+    <>
```

Comment:
ik theres nothing here yet but if you didnt already know, you don't need fragments if you already have a surrounding component like `<Container .. />`. They don't get rendered in the DOM but useful for grouping top level components as you can only return 1 element.